build.gradle

```diff
@@ -81,15 +81,6 @@ ospackage {
 
 def distributions = [
   [
-    "dist":        'openhab2-release',
```

Comment:
Does it make sense to remove them all together? What if someone wants to build their own version of 2.5.1 distro?

We can just change the job for 2.5.x builds on Jenkins.

Reply:
Yes, it imho makes sense for the HEAD of the 2.5.x branch. For building 2.5.1 distros, we should have a 2.5.1 tag at the appropriate version of the code that can be used for building that version.

Reply:
OK, makes sense I think. I need to find a fix for #154 before I can make a 2.5.1 tag for building the distro, but from what you're saying I will do this on master before making changes for 3.x